### ISSUE TITLE
The "Workspaces" applet/menu is a blank panel

### ISSUE THREAD

**Issue by nicolasdumitru** [pop-os/cosmic-epoch#785]
Hello! There is a bug that causes the "Workspaces" menu to be blank. None of my two workspaces (one of them is completely empty) is showing up in the menu. I tried both clicking the applet in the panel and opening it by using the default keybinding; the end result is the same (a picture of this bug is available below. It seems to me like it has to do with locking the screen and unlocking it a few times, but, unfortunately, I do not know what caused the bug and I cannot provide any meaningful algorithm for reproducing it.

![screenshot-2024-08-21-18-04-57](https://github.com/user-attachments/assets/074fb72c-fd10-490c-8e72-15f140cb3a07)

**Comment by leviport** [comment]
Are you using multiple displays? If so, this may be the same as https://github.com/pop-os/cosmic-workspaces-epoch/issues/61. If it happens on a single display setup, then it could still be related, but might not be the same bug.

**Comment by nicolasdumitru** [comment]
Yes, I am using two (mirrored) displays and it does seem to be related to that. I did not intend to make a duplicate bug report; I just didn't find the other one.

**Comment by leviport** [comment]
No worries, just wanted to link the two together if they were indeed related. I'll keep this open so people find it if they go to report it again, but I'll lock the thread so that further discussion happens in https://github.com/pop-os/cosmic-workspaces-epoch/issues/61. Thanks for the report!

**Comment by jacobgkau** [comment]
https://github.com/pop-os/cosmic-workspaces-epoch/issues/61 was resolved.
